docs/CHANGELOG.md

```diff
@@ -16,6 +16,16 @@ aliases:
 
 - [operator](#./README.md): fix VM CRs' `xxNamespaceSelector` and `xxSelector` options, previously they are inverted. See this [issue](https://github.com/VictoriaMetrics/operator/issues/980) for details.
 
+### Breaking changes
```

Comment:
Could you move this to the top of the changelog?

Reply:
done